core/platform/search/gae_search_services_test.py

```diff
@@ -704,3 +720,14 @@ def test_get_document_from_index(self):
             'my_doc', 'my_index')
         self.assertEqual(result.get('id'), 'my_doc')
         self.assertEqual(result.get('my_field'), 'value')
+
+
+class ClearIndexTests(test_utils.GenericTestBase):
+    def test_clear_index(self):
+        doc = {'id': 'doc1', 'k': 'abc def', 'rank': 3, 'language_code': 'en'}
+        gae_search_services.add_documents_to_index([doc], 'index')
+        result = gae_search_services.search('k:abc', index='index')[0]
+        self.assertEqual(result, [doc])
+        gae_search_services.clear_index('index')
+        result = gae_search_services.search('k:abc', index='index')[0]
+        self.assertNotEqual(result, [doc])
```

Comment:
Do self.assertEqual instead. Who knows, the result might be [doc, doc] -- this test is too weak.

Reply:
done